package.json

```diff
@@ -17,12 +17,14 @@
     "semantic-ui-react": "^0.88.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "CI= react-scripts build",
```

Comment:
The removal of this CI= seems to be causing the build to fail on Netlify. I'm going to try adding it back and seeing what happens.